src/HIndent/Ast/Name/Infix.hs

```diff
@@ -12,10 +12,11 @@ import qualified GHC.Unit.Module as GHC
 import HIndent.Ast.NodeComments
 import {-# SOURCE #-} HIndent.Pretty
 import HIndent.Pretty.Combinators
+import HIndent.Pretty.Combinators.Outputable (showOutputable)
```

Comment:
Weird that this shows up as redundant import. I see the symbol being used below.

I think this is because `showOutputable` is also exported from `HIndent.Pretty.Combinators` too: https://github.com/mihaimaruseac/hindent/blob/1563ceb303b95b078ca91d91233cae6dc26a48c7/src/HIndent/Pretty/Combinators/Outputable.hs#L7